README.md

```diff
@@ -117,3 +117,75 @@ npm publish --access public
 git push
 git push --tags
 ```
+
+
+### Packaging with vercel.pkg
```

Comment:
Change this to `### Releasing`.

And fix up the typos in the text. Put it into word/writer if necessary.

Reply:
Spacing of headers should be:

```
text text test

### Header

text text text
```